### PR TITLE
Add jade renderer

### PR DESCRIPTION
**Pull request by spinscale**
Using jade4j as an alternative renderer

### CODE REVIEW

**Comment by jknack** [comment]
love it!

will make some minor comments on your pull but thanks a lot for adding jade!!

File: jooby-jade/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# jade
```
Comment (jknack) [inline jooby-jade/README.md]: move this file to: ```md/jooby-jade```

the site generator will merge this file in the web site and add it here later.

File: jooby-jade/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+[Jade](http://jade-lang.com/) templates for [Jooby](/). Exposes a [renderer](/apidocs/Renderer.html).
+
+## dependency
+
+```xml
+<dependency>
+  <groupId>org.jooby</groupId>
+  <artifactId>jooby-jade</artifactId>
+  <version>0.14.0</version>
```
Comment (jknack) [inline jooby-jade/README.md]: replace ```0.14.0``` with ```{{version}}```

File: jooby-jade/README.md (hunk 1 above)
Comment (jknack) [inline jooby-jade/README.md]: sorry, correct location is: ```md/doc/jade``` (not ```md/jooby-jade```)

File: jooby-jade/src/test/java/org/jooby/jade/JadeTest.java [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+          unit.mockStatic(Multibinder.class);
+          expect(Multibinder.newSetBinder(binder, Renderer.class)).andReturn(formatter);
+        })
+        .run(unit -> {
+          new Jade()
+              .configure(unit.get(Env.class), unit.get(Config.class), unit.get(Binder.class));
+        });
+  }
+
```
Comment (jknack) [inline jooby-jade/src/test/java/org/jooby/jade/JadeTest.java]: can you add a test for ```Engine.toString()``` just verify name is ```jade```

**Comment by spinscale** [comment]
thx for your feedback, incorporated all of it!

**Comment by spinscale** [comment]
Also, feel free to rebase (I hate merge commits and PRs with many commits) :-)

**Pull request merged**